sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/ColumnVector.java

```diff
@@ -387,6 +449,11 @@ private void throwUnsupportedException(int requiredCapacity, Throwable cause) {
   public abstract byte getByte(int rowId);
 
   /**
+   * Gets values from [rowId, rowId + count) to [dst, dst + count)
```

Comment:
Let's also mention that, it's the caller's responsibility to make sure the given array has enough space.